tests/sources/fixtures/oracle/fixture.py

```diff
@@ -21,10 +21,10 @@
 RETRIES = 1
 BATCH_SIZE = 100
 
-USER = "admin"
+USER = "c##admin"
```

Comment:
Just curious what "c##" means.

Reply:
I think it's like C++, but from Microsoft

![image](https://github.com/elastic/connectors/assets/12238374/0f4cf958-debd-45ca-b596-938c11801bcc)

Reply:
@vidok seems like in newer multitenant-based Oracle versions, when you want to create a 'common' user (available in all containers), you have to prefix it with `c##`: 
> The username for the common user must be prefixed with "C##" or "c##" and contain only ASCII or EBCDIC characters.

We have to do this now because we are switching to a more modern version of Oracle.